arcsi/templates/item/add.html

```diff
@@ -30,7 +30,7 @@ <h2>{% block title %}Add episode to archive{% endblock %}</h2>
 </div>
 
   <div>
-    <label for="number">Episode number</label>
+    <label for="number">Episode number (+1 by def)</label>
```

Comment:
I would try to word this differently. Leave empty to auto-increment or some such.